opencti-platform/opencti-front/src/private/components/search/SearchStixCoreObjectLine.jsx

```diff
@@ -331,6 +331,11 @@ export const SearchStixCoreObjectLine = createFragmentContainer(
           description
           aliases
         }
+        ... on MalwareAnalysis {
```

Comment:
would it work to use representative everywhere ? by removing all these specific cases and using just : 
```
... on StixObject {
  representative {
    main
    secondary
  }
}
```

Reply:
I also think all this part is useless : 

```
... on IPv4Addr {
  countries {
    edges {
      node {
        name
        x_opencti_aliases
      }
    }
  }
}
... on IPv6Addr {
  countries {
    edges {
      node {
        name
        x_opencti_aliases
      }
    }
  }
}
```